pkg/apis/build/v1alpha2/builder_conversion.go

```diff
@@ -3,70 +3,70 @@ package v1alpha2
 import (
 	"context"
 	"fmt"
+
 	"knative.dev/pkg/apis"
 
 	"github.com/pivotal/kpack/pkg/apis/build/v1alpha1"
 )
 
-func (i *Builder) ConvertTo(_ context.Context, to apis.Convertible) error {
+func (b *Builder) ConvertTo(_ context.Context, to apis.Convertible) error {
 	switch toBuilder := to.(type) {
 	case *v1alpha1.Builder:
-		toBuilder.ObjectMeta = i.ObjectMeta
-		i.Spec.convertTo(&toBuilder.Spec)
-		i.Status.convertTo(&toBuilder.Status)
+		toBuilder.ObjectMeta = b.ObjectMeta
+
+		b.Spec.convertTo(&toBuilder.Spec)
+		b.Status.convertTo(&toBuilder.Status)
 	default:
 		return fmt.Errorf("unknown version, got: %T", toBuilder)
 	}
 	return nil
 }
 
-func (i *Builder) ConvertFrom(_ context.Context, from apis.Convertible) error {
+func (b *Builder) ConvertFrom(_ context.Context, from apis.Convertible) error {
 	switch fromBuilder := from.(type) {
 	case *v1alpha1.Builder:
-		i.ObjectMeta = fromBuilder.ObjectMeta
-		i.Spec.convertFrom(&fromBuilder.Spec)
-		i.Status.convertFrom(&fromBuilder.Status)
+		b.ObjectMeta = fromBuilder.ObjectMeta
+		b.Spec.convertFrom(&fromBuilder.Spec)
+		b.Status.convertFrom(&fromBuilder.Status)
 	default:
 		return fmt.Errorf("unknown version, got: %T", fromBuilder)
 	}
 
 	return nil
 }
 
-func (is *NamespacedBuilderSpec) convertTo(to *v1alpha1.NamespacedBuilderSpec) {
-	to.Tag = is.Tag
-	to.Stack = is.Stack
-	to.Store = is.Store
-	to.Order = is.Order
-	to.ServiceAccount = is.ServiceAccount()
+func (bs *NamespacedBuilderSpec) convertTo(to *v1alpha1.NamespacedBuilderSpec) {
+	to.Tag = bs.Tag
+	to.Stack = bs.Stack
+	to.Store = bs.Store
+	to.Order = bs.Order
+	to.ServiceAccount = bs.ServiceAccountName
```

Comment:
I believe we still want `ServiceAccount()` here so we can fall back to BackwardsCompatibleServiceAccount if necessary.